MainRepository/Program.cs

```diff
@@ -1,7 +1,23 @@
-﻿internal class Program
+﻿// Print chessboard

```

Comment:
Alignment is completely wrong. Use "Ctrl + K" & "Ctrl + F" to view the difference.
And I'm unable to compile.